{"version":3,"sources":["../../src/errors/internal-error.js"],"names":["app","info","err","log","error","Promise","reject","message"],"mappings":";;;;;;;;AAAA;;;;;;kBAEe,UAACA,GAAD,EAAoB;AAAA,MAAdC,IAAc,uEAAP,EAAO;;AACjC,SAAQC,GAAD,IAAS;AACdF,QAAIG,GAAJ,CAAQC,KAAR,CAAcF,GAAd,EAAmBD,IAAnB;AACA,WAAOI,QAAQC,MAAR,CAAe,gCAAQC,oCAAR,IAA0CN,IAA1C,EAAf,CAAP;AACD,GAHD;AAID,C","file":"internal-error.js","sourcesContent":["import error, { ERROR_INTERNAL_ERROR } from './error';\n\nexport default (app, info = {}) => {\n  return (err) => {\n    app.log.error(err, info);\n    return Promise.reject(error({ message: ERROR_INTERNAL_ERROR, ...info }));\n  };\n};"]}